### PR TITLE
Format type from % to f-string

### PR DESCRIPTION
**Pull request by shenjackyuanjie**
lol, just some typing change

### CODE REVIEW

**Comment by shenjackyuanjie** [comment]
actully. this PR was not finish

File: pyglet/canvas/xlib.py [hunk 1]
```diff
@@ -274,9 +274,7 @@ def restore_mode(self):
             self.set_mode(self._initial_mode)
 
     def __repr__(self):
-        return 'XlibScreen(display=%r, x=%d, y=%d, ' \
-               'width=%d, height=%d, xinerama=%d)' % \
-               (self.display, self.x, self.y, self.width, self.height, self._xinerama)
+        return f"{self.__class__.__name__}(display={self.display!r}, x={self.x}, y={self.y}, width={self.width}, height={self.height}, xinerama={self._xinerama})"
```
Comment (benmoran56) [inline pyglet/canvas/xlib.py:277]: Can you break the line at about 100 characters?

**Comment by benmoran56** [comment]
Thank you for working on this. Please comment again when you're ready, and I'll give it another look.

File: pyglet/canvas/xlib.py (hunk 1 above)
Comment (shenjackyuanjie) [reply pyglet/canvas/xlib.py:277]: got it， i will

**Comment by shenjackyuanjie** [comment]
> Thank you for working on this. Please comment again when you're ready, and I'll give it another look.

Thanks♪(･ω･)ﾉ

**Comment by shenjackyuanjie** [comment]
thanks @Zhuruoling for helping me

**Comment by benmoran56** [comment]
Hi, 

Is there still more work you want to do, or do you think it's ready to merge in?

**Comment by shenjackyuanjie** [comment]
> Hi,
> 
> Is there still more work you want to do, or do you think it's ready to merge in?

sorry, I just saw it
I will commit some today, If it passes the test, merge it please

**Review by benmoran56 — APPROVED** [review]

**Pull request merged**